rofl-appd/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rofl-appd"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+# Oasis SDK.
+oasis-runtime-sdk = { path = "../runtime-sdk", features = ["tdx"] }
+
+# Third party.
+anyhow = "1.0.86"
+rocket = { git = "https://github.com/rwf2/Rocket", rev = "28891e8072136f4641a33fb8c3f2aafce9d88d5b", features = ["json"] }
```

Comment:
Please add a short comment why a specific commit is used (and maybe a TODO to update to a release once the required features are released)

Reply:
Added.

Reply:
Thanks!